extensions/dicom-seg/src/components/SegmentationPanel.js

```diff
@@ -0,0 +1,473 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import cornerstoneTools from 'cornerstone-tools';
+import cornerstone from 'cornerstone-core';
+import moment from 'moment';
+
+import { utils } from '@ohif/core';
+import { ScrollableArea, TableList, Icon } from '@ohif/ui';
```

Comment:
It looks like we actually don't specify `@ohif/ui` as a dependency either, haha.

Reply:
Bundlers are weird.